pnpm-lock.yaml

```diff
@@ -5272,6 +5273,19 @@ packages:
       uuid: 3.4.0
     dev: false
 
+  /graphql-upload/13.0.0_graphql@14.5.8:
+    resolution: {integrity: sha512-YKhx8m/uOtKu4Y1UzBFJhbBGJTlk7k4CydlUUiNrtxnwZv0WigbRHP+DVhRNKt7u7DXOtcKZeYJlGtnMXvreXA==}
+    engines: {node: ^12.22.0 || ^14.17.0 || >= 16.0.0}
+    peerDependencies:
+      graphql: 0.13.1 - 16
+    dependencies:
+      busboy: 0.3.1
+      fs-capacitor: 6.2.0
+      graphql: 14.5.8
+      http-errors: 1.8.1
+      object-path: 0.11.8
+    dev: false
```

Comment:
Questions : Why are we using dev: false here ?

Reply:
Like how to say, I actually don't know why it's there, so for this, I am using the 2 packages 
1. graphql-upload and it's types
2. fs-capacitor
after adding these it automatically gets updated and I have committed it. But by my side I have not added it. It's got added via pnpm side. 

As a side check, I will try to re-generate pnpm-lock.yaml to cross-varify is that would be the same or not

Reply:
In simple words: when `dev: false` it means this dependency is required to run our application and has to be present in our node_modules when we run `pnpm install --prod`. And `dev: true` means it  is used when we develop our application but it is not required to install on our production environments.